src/Compilers/CSharp/Portable/CSharpParseOptions.cs

```diff
@@ -36,13 +36,13 @@ public override IEnumerable<string> PreprocessorSymbolNames
         }
 
         public CSharpParseOptions(
-            LanguageVersion languageVersion = LanguageVersion.CSharp7,
+            LanguageVersion languageVersion = LanguageVersion.Latest,
             DocumentationMode documentationMode = DocumentationMode.Parse,
             SourceCodeKind kind = SourceCodeKind.Regular,
             IEnumerable<string> preprocessorSymbols = null)
-            : this(languageVersion, documentationMode, kind, preprocessorSymbols.ToImmutableArrayOrEmpty())
+            : this(languageVersion.MapLatestToVersion(), documentationMode, kind, preprocessorSymbols.ToImmutableArrayOrEmpty())
         {
-            if (!languageVersion.IsValid())
+            if (!LanguageVersion.IsValid())
```

Comment:
Agree.  Feel like we should be checking the argument here directly vs. indirectly through the final property.